### PR TITLE
Add jqwidgets via single package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by DimitarD**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jqwidgets/package.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jqwidgets/*.*",
+        "jqwidgets/*/*.*",
+        "jqwidgets-ng/*.*",
+        "jqwidgets-ng/*/*.*",
+        "jqwidgets-ng/*/*/*.*",
+        "jqwidgets-ng/*/*/*/*.*",
```
Comment (MattIPv4) [inline ajax/libs/jqwidgets/package.json]: Why not just use `jqwidgets-ng/**/*.*`?

File: ajax/libs/jqwidgets/package.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    "type": "git",
+    "url": "git://github.com/jqwidgets/jQWidgets"
+  },
+  "npmName": "jqwidgets-scripts",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "jqwidgets/*.*",
+        "jqwidgets/*/*.*",
```
Comment (MattIPv4) [inline ajax/libs/jqwidgets/package.json]: Why not just use `jqwidgets/**/*.*`?

File: ajax/libs/jqwidgets/package.json [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    "Panel"
+  ],
+  "author": {
+    "name": "jQWidgets",
+    "email": "support@jqwidgets.com",
+    "url": "http://www.jqwidgets.com/"
+  },
+  "license": "http://www.jqwidgets.com/license/",
+  "homepage": "http://www.jqwidgets.com/",
+  "filename": ""
```
Comment (MattIPv4) [inline ajax/libs/jqwidgets/package.json]: Is there no default file that you can recommend users should use?

File: ajax/libs/jqwidgets/package.json (hunk 1 above)
Comment (DimitarD) [reply ajax/libs/jqwidgets/package.json]: Your feedback has been implemented.

File: ajax/libs/jqwidgets/package.json (hunk 2 above)
Comment (DimitarD) [reply ajax/libs/jqwidgets/package.json]: Your feedback has been implemented.

File: ajax/libs/jqwidgets/package.json (hunk 3 above)
Comment (DimitarD) [reply ajax/libs/jqwidgets/package.json]: Your feedback has been implemented.

**Pull request merged**